src/test/java/seedu/address/logic/commands/EditCommandTest.java

```diff
@@ -114,9 +119,10 @@ public void execute_duplicateStudentUnfilteredList_failure() {
 
     @Test
     public void execute_duplicateStudentFilteredList_failure() {
+
         showStudentAtIndex(model, INDEX_FIRST_STUDENT);
 
-        // edit student in filtered list into a duplicate in address book
+        // EP: editing a student to a duplicate in the ugteach data file
```

Comment:
I feel that original sentence sounds abit better?

Reply:
updated. Please take a look,